index.html

```diff
@@ -325,6 +327,10 @@ <h2><code>PointerEvent</code> interface</h2>
                         <dd>
                             <p>Indicates if the pointer represents the <a>primary pointer</a> of this pointer type.</p>
                         </dd>
+                    <dt><dfn>deviceId</dfn></dt>
+                        <dd>
+                            <p>A unique identifier for the device that generates the pointer event. Pointer events generated from the same physical device will have the same <code>deviceId</code> for the browsing session. User agents MAY reserve a generic <code>deviceId</code> value of <code>0</code> or <code>1</code> for events generated by the primary mouse device. The <code>deviceId</code> value of <code>-1</code> MUST be reserved and used to indicate events whose generating device could not be identified.</p>
```

Comment:
Should spec mention that the deviceId should be unique per browser session/document instance to avoid fingerprinting as mentioned in the [explainer](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/PointerEventDeviceId/explainer.md)?